src/modules/commander/airspeed_calibration.cpp

```diff
@@ -110,8 +110,7 @@ int do_airspeed_calibration(orb_advert_t *mavlink_log_pub)
 		param_get(param_find("SENS_DPRES_ANSC"), &(analog_scaling));
 		if (fabsf(analog_scaling) < 0.1f) {
 			calibration_log_critical(mavlink_log_pub, "[cal] No airspeed sensor, refer to the following:");
-			calibration_log_critical(mavlink_log_pub, "http://px4.io/docs/sensor-selection/");
-			calibration_log_critical(mavlink_log_pub, "http://px4.io/docs/vtols-without-airspeed-sensor/");
+			calibration_log_critical(mavlink_log_pub, "[cal] https://docs.px4.io/en/getting_started/sensor_selection.html");
```

Comment:
This is too long, 50 is the maximum, and the [cal] shoud be removed here as well.

Reply:
Thanks, I always miss the length.
I left [cal] for airspeed because there's no graphical calibration. All of this is dumped into the airspeed calibration window.

Reply:
I think we'll need to drop the URLs for now.